lib/form/paragraph3.flow

```diff
@@ -44,6 +44,9 @@ reconstructed on each reflow, because we combine them together to have as few sp
 possible, and because rendering words in the same sprite improves the interword spacing.
 */
 
+INF_METRICS = 100000.0;
```

Comment:
We have several places where different "infinite" width are used (innovation repor).
They looks like 10000.0 or similar. Please, try to find those places and change to use this common constant instead

Reply:
/lib/form/gui.flow line 403
InfiniteWidthAWH = const(WidthHeight(10000.0, 0.0))
?